solidity/deploy/01_deploy_tbtc_v2_token.ts

```diff
@@ -0,0 +1,21 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre
+  const { deploy } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  // This deployment is named TBTCTokenV2 to avoid conflict with TBTCToken
+  // deployment which represents the tBTC v1 token shipped via an external
+  // dependency.
+  await deploy("TBTCTokenV2", {
+    contract: "TBTCToken",
```

Comment:
Are we 100% sure it loads the `TBTCToken` from V2? In the `00_resolve_tbtc_v1_token.ts ` script, we load the same name of artifact from tbtc v1, are we sure it loads the correct artifact?
https://github.com/keep-network/tbtc-v2/blob/127232cb803a929d28a2e40115ca80a9ac3315f1/solidity/deploy/00_resolve_tbtc_v1_token.ts#L9

Reply:
I'm trying to figure out the best way we can handle that to be certain we don't mix them up. 

Maybe some post-install script that would rename `node_modules/@keep-network/tbtc/artifact/TBTCToken.json` to `node_modules/@keep-network/tbtc/artifact/TBTCTokenV1.json` right after yarn downloads the dependencies?

Any thoughts, ideas?